.azure-pipelines/azure-pipelines-linux.yml

```diff
@@ -10,8 +10,14 @@ jobs:
   strategy:
     maxParallel: 8
     matrix:
-      linux_:
-        CONFIG: linux_
+      linux_mpimpich:
+        CONFIG: linux_mpimpich
+        UPLOAD_PACKAGES: True
+      linux_mpinompi:
```

Comment:
linux_nompi seems to be less cluttered, if there is no strong reason to prefix every element with linux_mpi?

Reply:
This is an internal build file and has no relevance for the final namespace.

Reply:
Also this file is autogenerated by the conda-forge tooling so we absolutely cannot change it.